python/ray/serve/controller.py

```diff
@@ -8,6 +8,7 @@
 from typing import Any, Dict, Iterable, List, Optional, Tuple, Union
 
 import ray
+from ray.util import metrics
```

Comment:
@sihanwang41 should Shreyas be using the Serve-specific metrics wrapper?

Reply:
If no serve specific level information needed, either is fine to me.